multi/imxrt-multi/imxrt-multi.h

```diff
@@ -17,14 +17,20 @@
 #include <stdlib.h>
 
 /* IDs of special files OIDs */
+/* clang-format off */
 enum { id_console = 0, id_uart1, id_uart2, id_uart3, id_uart4, id_uart5, id_uart6, id_uart7, id_uart8,
 	id_uart9, id_uart10, id_uart11, id_uart12, id_gpio1, id_gpio2, id_gpio3, id_gpio4, id_gpio5,
 	id_gpio6, id_gpio7, id_gpio8, id_gpio9, id_gpio10, id_gpio11, id_gpio12, id_gpio13, id_spi1,
 	id_spi2, id_spi3, id_spi4,
 #ifdef TARGET_IMXRT1170
 	id_spi5, id_spi6,
 #endif
-	id_i2c1, id_i2c2, id_i2c3, id_i2c4, id_trng };
+	id_i2c1, id_i2c2, id_i2c3, id_i2c4, id_trng,
+#ifdef TARGET_IMXRT1170
+	id_cm4_0, id_cm4_1, id_cm4_2, id_cm4_3
```

Comment:
IDs are not available outside of phoenix-rtos-devices.

Reply:
Wym, they are available, but I agree shouldn't. Removed from API, lookup should be used